travis/maven-settings.xml

```diff
@@ -53,14 +46,27 @@
 
   <servers>
     <server>
-      <id>io.cloudrepo.rit-public-snapshot</id>
-      <username>${env.REPO_USERNAME}</username>
-      <password>${env.REPO_API_TOKEN}</password>
-    </server>
-    <server>
-      <id>io.cloudrepo.rit-ext-private</id>
-      <username>${env.REPO_USERNAME}</username>
-      <password>${env.REPO_API_TOKEN}</password>
+      <id>artifact-registry</id>
+      <configuration>
+        <httpConfiguration>
+          <get>
+            <usePreemptive>true</usePreemptive>
+          </get>
+          <head>
+            <usePreemptive>true</usePreemptive>
+          </head>
+          <put>
+            <params>
+              <property>
+                <name>http.protocol.expect-continue</name>
+                <value>false</value>
+              </property>
+            </params>
+          </put>
+        </httpConfiguration>
+      </configuration>
+      <username>_json_key_base64</username>
+      <password>${env.ARTIFACT_REGISTRY_KEY}</password>
```

Comment:
In the other build I replaced a token for the credential in the file before the main build step, and cleared the environment variable. Just a bit of extra paranoia to avoid accidental exposure of environment variables.